data/json/items/comestibles.json

```diff
@@ -2422,12 +2422,12 @@
         "symbol" : "%",
         "quench" : -1,
         "healthy" : 1,
-        "nutrition" : 40,
+        "nutrition" : 24,
```

Comment:
I'll double check vs salted, but here's what I said in a previous revision

>After reviewing its clear that the volume, nutrition, and vitamins of fish may be off and require its own audit.

>The key factor for me here is ensuring the final volume of the products matches the jars, and since this is the volume ratio that was previously in place (and matches all the existing recipes) I'm not willing to change it in this particular fix.

>I do think it's broken and needs a fix, it's just outside the scope of this; since it would require changing every recipe that uses pickled fish, and there are questions about what the resulting volume should be given the volume of the source materials.